src/main/webapp/app/entities/lecture/lecture.module.ts

```diff
@@ -2,15 +2,17 @@ import { NgModule } from '@angular/core';
 import { RouterModule } from '@angular/router';
 
 import { ArtemisSharedModule } from 'app/shared';
-import { LectureAttachmentsComponent, LectureComponent, LectureDetailComponent, lectureRoute, LectureUpdateComponent } from './';
+import { LectureAttachmentsComponent, LectureComponent, LectureDetailComponent, lectureRoute, LectureService, LectureUpdateComponent } from './';
```

Comment:
I think we do not need the `LectureService`.

```suggestion
import { LectureAttachmentsComponent, LectureComponent, LectureDetailComponent, lectureRoute, LectureUpdateComponent } from './';
```